src/CmdLine.hs

```diff
@@ -174,6 +175,7 @@ mode = cmdArgsMode $ modes
         ,cmdCppSimple = nam_ "cpp-simple" &= help "Use a simple CPP (strip # lines)"
         ,cmdCppAnsi = nam_ "cpp-ansi" &= help "Use CPP in ANSI compatibility mode"
         ,cmdJson = nam_ "json" &= help "Display hint data as JSON"
+        ,cmdJsonCC = nam_ "json-cc" &= help "Display hint data as Code Climate Issues JSON"
```

Comment:
Should the flag just be --code-climate? The important thing is it's in code climate format. The fact code climate format is JSON is just a detail.

Reply:
`--code-climate` works for me. `--cc` would also be OK with me, I'm not sure if there's risk in the brevity, WDYT?